src/components/OrgUpdate/OrgUpdate.tsx

```diff
@@ -45,7 +45,7 @@ function OrgUpdate(props: OrgUpdateProps): JSX.Element {
     try {
       const { data } = await login({
         variables: {
-          id: currentUrl,
+          id: props.id,
```

Comment:
@akankshat05 I didn't get why you've to change this it will break the functionality ;)